src/main/java/cooper/meetings/MeetingManager.java

```diff
@@ -0,0 +1,21 @@
+package cooper.meetings;
+
+import cooper.ui.Ui;
+
+import java.util.HashMap;
+
+public class MeetingManager {
+    private static HashMap<String, String> meetings = new HashMap(10);
```

Comment:
Is the size fixed at 10 for now?

Reply:
yep for now!! im thinking of having a fixed number of keys corresponding to the timings later